src/core.js

```diff
@@ -177,6 +177,9 @@ JSONEditor.prototype = {
     });
 
     if(!classname) throw "Unknown editor for schema "+JSON.stringify(schema);
+    if(classname === "array" && JSONEditor.plugins.selectize.enable) {
+        classname = classname + "Selectize";
+    }
```

Comment:
Can this be done with the resolver functions instead?  I don't like having this logic in core.js